op-deployer/cmd/op-deployer/main.go

```diff
@@ -44,6 +45,12 @@ func main() {
 			Flags:  cliapp.ProtectFlags(deployer.ApplyFlags),
 			Action: deployer.ApplyCLI(),
 		},
+		{
+			Name:        "upgrade",
+			Usage:       "upgrades contracts by sending tx to OPCM.upgrade function",
```

Comment:
It just outputs a calldata string though right?

Reply:
it supports multiple modes - calldata and calling directly.

Reply:
Could reword to "creates tx for OPCM.upgrade function, then either sends the tx onchain or prints calldata"